Bugsnag.PCL/Extensions/Extensions.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text.RegularExpressions;
+
+namespace Bugsnag.PCL.Extensions
+{
+    static class Extensions
+    {
+        /// <remarks>Not sure I'm really thrilled with this...</remarks>
+        public static IEnumerable<Exception> Unwrap(this Exception ex)
+        {
+            if (ex == null)
+            {
+                return Enumerable.Empty<Exception>();
+            }
+            else if (ex.InnerException == null)
+            {
+                return new Exception[] { ex };
+            }
+
+            return ex.InnerException.Unwrap().Concat(new Exception[] { ex });
+        }
+
+        public static IEnumerable<string> ToLines(this Exception ex)
+        {
+            if (ex == null || ex.StackTrace == null)
+            {
+                return new string[] { String.Empty };
```

Comment:
It is possible to make the replacement, but need to check if Bugsnag will register that event. I think there should be at least one line to make it work properly